gtsfm/configs/deep_front_end.yaml

```diff
@@ -14,6 +14,7 @@ SceneOptimizer:
 
   two_view_estimator:
     _target_: gtsfm.two_view_estimator.TwoViewEstimator
+    bundle_adjust_2view: True
```

Comment:
Is it possible to also expose the number of BA iterations?

Reply:
I have removed the number of iterations as not changing the value does not lead to a significant time difference.

Reply:
Do you mean to say that reducing the number of iterations did not result in a significant time difference? If so, could you support this with some runtime values? 

From what I remember, reducing the number of iterations noticeably reduced runtimes.

Reply:
This is from the door dataset on my local machine.

2-view BA with no max iters:
- Average: 1.72
- Median: 1.24


2-view BA with 20 max iters:
- Average: 1.20
- Median: 1.14

From the CIs
- [Without max iters](https://github.com/borglab/gtsfm/actions/runs/1511043825): 2 hrs 27 mins
- [With 20 max iters](https://github.com/borglab/gtsfm/actions/runs/1510032128): 2 hours 10 mins

I feel the difference is not prohibitive to have one more hyperparam.

Reply:
Average statistics can be skewed by some outliers, which can be caused by some random fluctuations. The median on door-12 on my machine is just off by 8%. The CI run is off by 13%, and that includes a pretty big dataset.

I agree that we will save some compute by having a cap on the number of iterations, but I have two concerns about having a cap:
1. When will we use it? In the CI we would want to achieve the best performance for now. In the future we might revisit this problem when we have a dataset which is huge relative to skydio-501. For the local runs, I think we will either use BA or not use BA. I don't see a reason when we would want to fiddle with the hyperparam to achieve ~10% 2-view BA compute savings.
2. What is the best cap on the number of iterations?


@dellaert @johnwlambert @akshay-krishnan for your opinions too.